src/handlers/major_change.rs

```diff
@@ -587,10 +588,10 @@ async fn process_seconded(
             .post_comment(
                 &ctx.github,
                 &*format!(
-                    "The final comment period is now complete, this major change is now **accepted**.\
-                    \
-                    As the automated representative, I would like to thank the author for their work and everyone else who contributed to this major change proposal.\
-                    \
+                    "The final comment period is now complete, this major change is now **accepted**.\n\
```

Comment:
Using a `r"` string would be a bit nicer, to avoid the `\` newline escapes.

Reply:
Though of it, but the at the same time it shifts the content way to the left which isn't nice.

Reply:
Putting every lines to the far left is better, pushed that.